### PR TITLE
Add Apache 2.0 LICENSE

### LINKED ISSUE #12: Add Licence file

### PR DESCRIPTION
**Pull request by juliaogris**
Fixes #12 

#### Change proposed in this PR:

Add standard Apache 2.0 License

### CODE REVIEW

**Review by anzboi — DISMISSED** [review]

**Comment by camscale** [comment]
I've left reviewing this to @anzdaddy as I think he is in the best place to approve the license.

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (anzboi) [inline LICENSE]: Don't know what `[yyyy] [name of copyright owner]` should be replaced with.

Comment (juliaogris) [reply LICENSE]: OUPS :)

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright 2019 ANZ Bank - Australia and New Zealand Banking Group Limited
```
Comment (anzdaddy) [inline LICENSE]: "ANZ Bank" is not a formal name for ANZ. Write the following instead:

```
   Copyright 2019 Australia and New Zealand Banking Group Limited
```

**Review by anzdaddy — CHANGES_REQUESTED** [review]

File: LICENSE (hunk 2 above)
Comment (juliaogris) [reply LICENSE]: PTAL @anzdaddy

**Review by anzdaddy — APPROVED** [review]

**Pull request merged**